run3/flow/compute_efficiency.py

```diff
@@ -1,9 +1,14 @@
+import ctypes
```

Comment:
We should integrate this with the pt-weight computation, am I right?

Reply:
loading the pT weights from configuration